src/auth0-sso-login.js

```diff
@@ -155,9 +198,6 @@ export default class auth {
       webAuth.renewAuth(renewOptions, (err, authResult) => {
         if (err) {
           this.log(`Failed to update ID token on retry ${retries}: ${JSON.stringify(err)}`);
-          if (err.error === 'login_required') {
```

Comment:
If the SSO session in Auth0 has been removed, it means that they probably have logged out of either client of this library or by calling the `https://auth0.com/logout` url, shouldn't we force the user to login again?

Reply:
We still do. I moved it out of here, as this is one specific case and we were most likely missing some other (and not removing login).

Just follow the calls in `ensureLoggedIn` and you will find it. The sequence there is easy:
1. Try silent `renewAuth` -> failure
2. Call `removeLogin`, so that client has a chance to react to auth0 lock open and no valid token available (e.g. stop scheduled calls which use token)
3. Open auth0 lock and force login

Reply:
I see, missed that.